bitbots_bringup/config/camera_settings.yaml

```diff
@@ -109,11 +109,11 @@ gamma: 1.0
 #  with the MTU size the network card supports. A value greater 3000
 #  should be good (1500 for RaspberryPI)
 # gige:
-gige/mtu_size: 1500
+gige/mtu_size: 3000
```

Comment:
The changes in this file do not belong to this PR (and have already been updated on master today), please remove them.